gateway/api/views/programs.py

```diff
@@ -205,6 +206,13 @@ def run(self, request):
                 )
                 return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+            if settings.MAINTENANCE:
```

Comment:
Mmm... I don't know if I agree with this. I think even if the scheduler is in maintenance mode we should continue letting create jobs. Those jobs are going to be QUEUED but that's not problematic meanwhile we don't process them, WDYT @korgan00 ?